Dev/src/study_tools/cli_chat.py

```
@@ -0,0 +1,43 @@
+"""CLI chat interface."""
+
+import argparse
+from pathlib import Path
+
+# heavy imports done in main()
+
+from .utils import load_config
+
+
+def main():
```

Pylint (3.3.7)

[refactor] Too many local variables (16/15)

(R0914)